openmm-rbfe/ApplyingProtocolToNetworkQuickrunDemo.ipynb

```diff
@@ -0,0 +1,520 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "edges = list(network.edges)\n",
+    "\n",
+    "edges[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cd9e92f",
+   "metadata": {},
+   "source": [
+    "## 2) Define a Protocol and apply to a Network\n",
+    "\n",
+    "Next we will define a free energy protocol (method) and apply this to all edges in our free energy network\n",
+    "The `openfe.protocols` submodule contains many prepared workflows for estimating ,\n",
+    "here we import an RBFE method which uses the OpenMM MD engine.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "    <b>Note:</b>\n",
+    "    \n",
+    "The `openfe` package will contain a set of workflows,\n",
+    "and other academic groups are currently in the process of writing their own workflows.\n",
```

Comment:
we do say "will contain" so it's future tense, and indicating other groups are working in the same API doesn't have to mean it's importable via openfe

The point I'm trying to make here is that learning this API is useful as it's going to be a common interface to many methods.

Re: Ivan's stuff, it's not impossible that we allow importing this via openfe, (and separately that we move `openmm_rbfe` to a separate repo) since openfe is mostly just pulling in other resources under one roof.  Packaging is a discussion to be had though.

Reply:
It's the "currently" that's a problem here. That's no longer future tense

Reply:
Semantics we can argue over later.